scripts/distro-a.env

```diff
@@ -0,0 +1 @@
+SPA_CONFIG_URLS=/openmrs/spa/configs/ozone-frontend-config.json,/openmrs/spa/configs/ozone-haiti-frontend-config.json,/openmrs/spa/configs/ozone-hsc-frontend-config.json
```

Comment:
@VaishSiddharth, **if** the `SPA_CONFIG_URLS` is being passed into the O3 build config, these need to be separated by `;` instead of `,`, unless you are splitting them by `,` and then joining by `;` before passing into the build config.
Thanks!

Reply:
Are you sure about this @VaishSiddharth , I think we've always used `,` so far and it worked.